.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,17 @@
+# Ultralytics 🚀 AGPL-3.0 License - https://ultralytics.com/license
+
+blank_issues_enabled: true
```

💭 **SUGGESTION**: `blank_issues_enabled: true` allows users to bypass the structured templates, which may reduce the quality of incoming reports. If the goal is to strongly encourage use of these new forms, consider setting this to `false` so reporters must pick a template or a contact link.

**Suggested change:**
```suggestion
blank_issues_enabled: false
```